src/select/types.js

```diff
@@ -29,25 +29,25 @@ export type ParamsT = {
 export type OverridesT = {
   Root?: OverrideT<*>,
   Input?: OverrideT<*>,
-  SearchIcon?: OverrideT<*>,
+  SelectComponentIcon?: OverrideT<*>,
   InputContainer?: OverrideT<*>,
   Tag?: OverrideT<*>,
   Option?: OverrideT<*>,
   DropDown?: OverrideT<*>,
-  SearchIcon?: OverrideT<*>,
+  SelectComponentIcon?: OverrideT<*>,
   DropDownItem?: OverrideT<*>,
   SingleSelection?: OverrideT<*>,
 };
 
 export type OverridesDropDownT = {
   Option?: OverrideT<*>,
   DropDown?: OverrideT<*>,
-  SearchIcon?: OverrideT<*>,
+  SelectComponentIcon?: OverrideT<*>,
   DropDownItem?: OverrideT<*>,
 };
 
 export type PropsT = {
```

Comment:
I think you also want to remove `textValue` from PropsT?

Reply:
ok